### PR TITLE
Revendoring to latest gardener and ci related changes

### LINKED ISSUE #50: [Feature] Automatic Update-PR script
**Feature (What you would like to be added):**
This repository should also benefit from automatic update PRs of dependent components. etcd-druid deploys etcd-backup-restore, hence, when a new version of it is released then automatic update PRs should be opened by CI, similar to https://github.com/gardener/gardener/pull/2260.

**Motivation (Why is this needed?):**
Less manual actions.

**Approach/Hint to the implement solution (optional):**
You need such a script: https://github.com/gardener/gardener/blob/master/hack/.ci/set_dependency_version
You don't need to copy it but can also call it like the extensions as you already vendor `gardener/gardener`: https://github.com/gardener/gardener-extension-provider-aws/blob/master/.ci/set_dependency_version, https://github.com/gardener/gardener-extension-provider-aws/blob/master/hack/tools.go#L23

### PR DESCRIPTION
**Pull request by georgekuruvillak**
**What this PR does / why we need it**:
This PR revendors etcd-druid to use g/g v1.3.1. It updates the ci scripts.
**Which issue(s) this PR fixes**:
Fixes #50 
**Special notes for your reviewer**:

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

File: .ci/component_descriptor [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```
Comment (ialidzhikov) [inline .ci/component_descriptor:1]: File is not executable
```
chmod +x .ci/component_descriptor
```

File: .ci/set_dependency_version [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+"$(dirname $0)"/../vendor/github.com/gardener/gardener/hack/.ci/set_dependency_version
```
Comment (ialidzhikov) [inline .ci/set_dependency_version:3]:
```
$ ls -la vendor/github.com/gardener/gardener/hack/.ci
ls: cannot access 'vendor/github.com/gardener/gardener/hack/.ci': No such file or directory
```

You need to run `make revendor`.

Comment (georgekuruvillak) [reply .ci/set_dependency_version:3]: Done

File: .ci/component_descriptor (hunk 1 above)
Comment (georgekuruvillak) [reply .ci/component_descriptor:1]: Done

**Pull request merged**